### DISCUSSION TITLE
Why is my Link function grayed out?

### DISCUSSION THREAD

**Discussion by Bashbeazz** [visualboyadvance-m/visualboyadvance-m#1169] [Q&A] [answered]
Hey guys,

few days ago i startet using VBadvance. I am playing Pokemon Fire Red right now and started to think about exchange my pokemon. 
I tried it with VBA Link but it doesn't worked well cause of permanent disconnects and crashes. I saw the "normal" VBA has a link function too. I tried to start it but it is grayed out. Any help?

![image](https://github.com/visualboyadvance-m/visualboyadvance-m/assets/36849615/a39d3652-4c09-47ef-8b0e-cb34c796f4fb)

**Comment by Squall-Leonhart** [comment, marked as answer]
"Local Mode"

**Reply by Bashbeazz** [reply]
Thanks... for my understanding: I can just start 2 instances of the VB and connect them?
